serverless-configs/aws-infrastructure-resources.yml

```diff
@@ -223,3 +223,19 @@ Outputs:
         - Endpoint.Port
     Export:
       Name: ${self:provider.stage}-DatabasePort
+
+  VisualizeElastiCacheEndpoint:
```

Comment:
Don't use "Visualize" prefixes, that was the name of an entirely different project.